content/backend/graphql-js/4-adding-a-database.md

```diff
@@ -7,189 +7,238 @@ answers: ["To increase performance of the server", "When using two APIs, the Gra
+### Setting up our project with Prisma and SQLite 
 
-The Prisma server provides the _data access layer_ in your application architecture, making it easy for your API server to talk to the database through Prisma. The API of the Prisma server is consumed by the Prisma client inside your _API server_ implementation (similar to an ORM). The API server is what you've started building throughout the previous chapters using `graphql-yoga`.
+<Instruction>
 
-In essence, Prisma lets you easily connect the GraphQL resolvers in your API server with your database.
+First, let's install the Prisma CLI by running the following command in your terminal:
 
-### Setting up Prisma with a demo database
+```bash(path=".../hackernews-node/")
+npm install @prisma/cli --save-dev
+```
 
-In this tutorial, you're going to build everything entirely from scratch! For your Prisma configuration, you're going to start with the most minimal setup that's possible.
+</Instruction>
 
-The first thing you need to do is create two files, which you're going to put into a new directory called `prisma`.
+Then, update your project structure.
 
 <Instruction>
 
-First, create the `prisma` directory and then two files called `prisma.yml` and `datamodel.prisma` by running the following commands in your terminal:
+Create the `prisma` directory and then a file called `schema.prisma` by running the following commands in your terminal:
 
 ```bash(path=".../hackernews-node/")
 mkdir prisma
-touch prisma/prisma.yml
-touch prisma/datamodel.prisma
+touch prisma/schema.prisma
 ```
 
 </Instruction>
 
-[`prisma.yml`](https://www.prisma.io/docs/-5cy7/) is the main configuration file for your Prisma setup. `datamodel.prisma` on the other hand contains the definition of your [datamodel](https://www.prisma.io/docs/-knul). The Prisma datamodel defines your application's _models_. Each model will be mapped to a table in the underlying database.
+Remember the GraphQL schema that you've been working with until now? Well, Prisma has a schema, too! You can think of the `prisma.schema` file as a *database schema*. It has three components:
+
+1. **Data source**: Specifies your database connection.
+1. **Generator**: Indicates that you want to genenerate Prisma Client.
+1. **Data model**: Defines your application *models*. Each model will be mapped to a table in the underlying database.
+
+Prisma's unique data model bridges the gap to help you reason about your data in a way that maps very well to the underlying database, while still providing an abstraction that allows you to be productive with type safety and auto-completion.
 
-So far, the datamodel for your Hacker News app only contains one data type: `Link`. Because Prisma uses [GraphQL SDL](https://www.prisma.io/blog/graphql-sdl-schema-definition-language-6755bcb9ce51) for model definitions, you can basically copy the existing `Link` definition from `schema.graphql` into `datamodel.prisma`.
+Let's see it in action with your project!
```

Comment:
@nikolasburk I wonder if it is worth at some point mentioning the Prisma VS Code extension? With Journaly, I was recently very impressed with certain things such as how it even has the ability to automatically add in naming both sides of a relationship, etc. and it's pretty darn powerful! Even just the auto-formatting is very nice.

I could see why we would not mention it. It's just a thought if we might want to mention that certain editors have a great extension you can install for even more productivity benefits, syntax highlighting, etc.

Reply:
Yes, absolutely! Let's definitely mention the VS Code extension (and the equivalents for Jetbrains IDEs: https://plugins.jetbrains.com/plugin/14240-prisma)

Reply:
Ok got it!! 🙌🏼